db/repair_bucket.go

```diff
@@ -0,0 +1,228 @@
+package db
+
+import (
+	"encoding/json"
+
+	"io/ioutil"
```

Comment:
Imports need formatting.

Reply:
I ran goimports, no change.  Hmm...

Reply:
It's just the empty lines between the standard library imports that need to be removed, I think.